### DISCUSSION TITLE
Problem in the cover

### DISCUSSION THREAD

**Discussion by dansyndicate** [joaomlourenco/novathesis#478] [Q&A] [answered]
Good night professor,

I am having the following problem, in the cover before the "adviser:" it is written the setence "(en)" and i don't understand why.

I am using the recent file that you published recently for the "novathesis.cls" file.

I send to you the code from the file "3_cover.tex" to help find the error.

![image](https://github.com/joaomlourenco/novathesis/assets/129518837/c3bc9574-80ca-48d6-9b31-ddd3ebb2576b)
`%!TEX root = ../template.tex
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Config/3_cover.tex
%% NOVA thesis configuration file
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\typeout{NT FILE Config/3_cover.tex}%

%%==================================================================
%%  THESIS/DISSERTATION COVER DATA
%%==================================================================

%-------------------------------------------------------------------
% Thesis/Dissertation title (valid for all languages unless redefined as below)
% SYNTAX: \nttitle(type,lang){title}
%         For [type] use: [main]  for main title
%                         [sub]   for sub-title
%                         [spine] for a book spine version of the title
%                                 (if omittted defaults to main title without line breaks)
%-------------------------------------------------------------------
%         You may use "\\" to force line breaks
\nttitle(main,en){Explore a “Sub 0.7V Single Stage CTLE  \\with low power and double zero for equalization”}%
\nttitle(main,pt){Um Título de Tese Longo\\e com uma Mudança de Linha Forçada}%

%         You may use "\\" to force line breaks
\nttitle(sub,en){Preparation for the Dissertation}%
\nttitle(sub,pt){Algumas considerações sobre a Vida,\\o Universo e Tudo o Mais}%

%         You may NOT use "\\" to force line breaks
% \nttitle(spine,en){A Very Long and Impressive Thesis Title}%
% \nttitle(spine,pt){Um Título de Tese Longo e Impressionante}%

%-------------------------------------------------------------------
% DEPARTMENT NAME customization
% SYNTAX: \ntdepartment(language){name}
%         You may use "\\" to force line breaks
%         The stared (*) version will only define if undefined, otherwise ignored
% \/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
%    For FCT-NOVA, please open file “9_nova_fct.tex” and uncomment
%    the right name for your Department from the given list
%    otherwiese fill the entries below
%-------------------------------------------------------------------
\ntdepartment*(pt){Departamento de\\Nome do Departamento}
\ntdepartment*(en){Department of\\Electrical and Computer Engineering}

%-------------------------------------------------------------------
% DEGREE NAME customization
% SYNTAX: \ntdegreename(language){degree_name}
%         You may use "\\" to force line breaks
%         The stared (*) version will only define if undefined, otherwise ignored
% \/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
%    For FCT-NOVA, please open file “9_nova_fct.tex” and uncomment
%    the right name(s) for the PhD/MSc program from the given list
%    otherwiese fill the entries below
%-------------------------------------------------------------------
\ntdegreename*(pt){Nome do Programa de Estudos}
\ntdegreename*(en){Electrical and Computer Engineering}

%-------------------------------------------------------------------
% DEGREE SPECIALIZATION customization
%   If the degree includes a specialization, otherwise comment the
%   lines below
% SYNTAX: \ntspecialization(language){name}
%         You may use "\\" to force line breaks
%         The stared (*) version will only define if undefined, otherwise ignored
%-------------------------------------------------------------------
% \ntspecialization*(pt){Designação da Especialidade}
% \ntspecialization*(en){Speciality Name}

%-------------------------------------------------------------------
% SPONSORS customization
%   Some Schools require the work sponsors to be listed in the cover
% SYNTAX: \ntsponsors(language){names}
%-------------------------------------------------------------------
% \ntsponsors(en,1,logo){\includegraphics[height=1.5cm]{NOVAthesisFiles/Images/republica_portuguesa_CTES}}
% \ntsponsors(en,2,logo){\includegraphics[height=1.5cm]{NOVAthesisFiles/Images/FCT_MEC_logo_positive}}
% \ntsponsors(en,1,text){Funding Agency}

% \ntsponsors(pt,1,logo){\includegraphics[height=1.5cm]{NOVAthesisFiles/Images/republica_portuguesa_CTES}}
% \ntsponsors(pt,2,logo){\includegraphics[height=1.5cm]{NOVAthesisFiles/Images/FCT_MEC_logo_positive}}
% \ntsponsors(pt,1,text){Entidade Financiadora}

%-------------------------------------------------------------------
% AUTHOR customization
%    Author gender, full name, abd abbreviated name
% SYNTAX: \ntauthorname(gender){full name}{abbreviated name}
%         Use [f] for female and [m] for male
%-------------------------------------------------------------------
\ntauthorname(m){Daniel Vieira Martins}{Daniel Martins}

%-------------------------------------------------------------------
% AUTHOR previous degree
% SYNTAX: \ntauthordegree{degree}
%-------------------------------------------------------------------
\ntauthordegree(en){BSc in Electrical and Computer Engineering}
\ntauthordegree(pt){Mestrado/Licenciatura em Nome do Grau Anterior}

%-------------------------------------------------------------------
% SUBMISSION/EXAM DATE
% SYNTAX: \ntdate(submisison){year-month-day}       \ntdate(exam){year-month-day}
%         If <month> is a number, it will be converted to the month
%                   name in the cover language.
%-------------------------------------------------------------------
%Replay with your submisison and exam dates in format YYYY-MM-DD
\ntdate(submission){2024-02-11}
%\ntdate(exam){2024-10-11}

%-------------------------------------------------------------------
% ADVISERS LIST
% Advisers' gender and info (name, position, school)
% SYNTAX: \ntaddperson{adviser}(role,gender){Name, Position, School}
%         For [role] use: [a] for adviser
%                         [c] for co-adviser
%         For [gender] use: [f] for female
%                           [m] for male
%         You may use "\\" to force line breaks
% Advisers(s)
\ntaddperson{adviser}(a,m){Hugo Ribeiro, Eletronics Enginner, Synopsys Portugal}
% Co-advisers(s)
\ntaddperson{adviser}(c,m){Luís Oliveira, Associate Professor with Aggregation, NOVA University Lisbon}
%\ntaddperson{adviser}(c,f){John Doe other Co-Adviser Name, Full Professor, NOVA University Lisbon}

%-------------------------------------------------------------------
% COMMITTEE MEMBERS
% SYNTAX: \ntaddperson{committee}(role,gender){Name, Position, School}
%         For [role] use: [c] for committee chair
%                         [r] for rapporteurs
%                         [a] for advisers
%                         [ca] for co-advisers
%                         [m] for remaining members
%                         [g] for guests
%         For [gender] use: [f] for female
%                           [m] for male
%         You may use "\\" to force line breaks
%\ntaddperson{committee}(c,m){Name of the committee chairperson, Full Professor, FCT-NOVA}
%\ntaddperson{committee}(r,m){Name of a rapporteur, Associate Professor, Another University}
% \ntaddperson{committee}(r,m){Name of another rapporteur, Assistant Professor, Another University}
% \ntaddperson{committee}(a,m){Name of the adviser present in defense, Associate Professor, Some University}
% \ntaddperson{committee}(ca,m){Name of the co-adviser present in defense, Associate Professor, Some University}
%\ntaddperson{committee}(m,f){Another member of the committee, Full Professor, Another University}
%\ntaddperson{committee}(m,f){Yet another member of the committee, Assistant Professor, Another University}
% \ntaddperson{committee}(g,m){Guest member of the committee, Position, University}
`

**Comment by joaomlourenco** [comment, marked as answer]
That was a bug. Fixed. 
Just get the new `novathesis.cls`file and replace yours.

**Reply by dansyndicate** [reply]
Thank you professor.
